opm/simulators/linalg/mixed/bslv.c

```diff
@@ -0,0 +1,260 @@
+#define _POSIX_C_SOURCE 200809L // required for posix_memalgin in <stdlib.h>
```

Comment:
Any chance we can get rid of this in favor of a compiler argument or requesting a certain C standard?